visualize

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
```

Comment:
Easily doable -- the meat of this entire script is `chart_data = top_actor.get_orgchart()` followed by injecting that data into an HTML template.